subtractFlats.py

```diff
@@ -115,80 +104,93 @@ def main():
     suffix       = '_f2_R22_S11_E000.fits.gz'
 
     # Process Files
-    magnitude = ['18', '15', '14', '13', '12', '10']
+    magnitude = [18, 15, 14, 13, 12, 10]
     extraId   = ['0', '1', '2', '3', '4']
 
```

Comment:
I've learned a lot of interesting python tidbits from your changes!  

Why did you change the magnitudes from strings to integers but not the extraId here?